casr/src/bin/casr-cluster.rs

```diff
@@ -79,12 +96,19 @@ fn make_clusters(inpath: &Path, outpath: Option<&Path>, jobs: usize) -> Result<u
             if let Ok(trace) = stacktrace(casreps[i].as_path()) {
```

Comment:
You should inline code of `stacktrace` function here in order to avoid double reading from file